make/testing_flags.mk

```diff
@@ -65,4 +65,4 @@ else
 ITEST_TAGS += btcd
 endif
 
-ITEST := rm lntest/itest/output*.log; date; $(GOTEST) ./lntest/itest -tags="$(ITEST_TAGS)" $(TEST_FLAGS) -logoutput
+ITEST := rm lntest/itest/output*.log; rm lntest/itest/pprof*.log; date; $(GOTEST) ./lntest/itest -tags="$(ITEST_TAGS)" $(TEST_FLAGS) -logoutput -goroutinedump
```

Comment:
I think we'll also want to upload the goroutine dumps along with the logs in our travis builds.

Reply:
Since they end with `*.log` they will :)